slu-status/src/slu-status/slu-status.service.spec.ts

```diff
@@ -71,15 +76,18 @@ describe('StatusService', () => {
 		expect(result.toJSON()).toEqual(update);
 	});
 
-	it('should be null: no slu under this id', async () => {
+	it('should throw DB error: no slu under this id', async () => {
 		await service.saveSluStatus(body);
 		const update = {
 			id: 'id:iota:1223455',
 			status: Status.INSTALLED,
 			channelAddress: '186ae31cffc392c8de858b95e82591368fee453da41653469a35d442c18a4f7e0000000000000000:24268d0b046f16be9c169c3e'
 		};
-		const result = await service.updateSluStatus(update.id, update.status);
-		expect(result).toBeNull();
+		try {
+			await service.updateSluStatus(update.id, update.status);
+		} catch (err) {
+			expect(err.message).toBe('Could not find document in the collection.');
+		}
 	});
 
 	it('should return status', async () => {
```

Comment:
Maybe this line is missing?
```
const getSavedSluStatus = await service.getSluStatus('...');
```
Where is defined `getSavedSluStatus`?

Reply:
I don't understand. The line is present on local.